JASP-Engine/JASP/R/anovarepeatedmeasures.R

```diff
@@ -2501,6 +2527,273 @@ AnovaRepeatedMeasures <- function(dataset=NULL, options, perform="run", callback
 	list(result=descriptivesPlotList, status=status, stateDescriptivesPlot=stateDescriptivesPlot)
 }
 
+.rmAnovaSimpleEffects <- function(dataset, options, perform, fullModel, fullAnovaTableWithin, fullAnovaTableBetween, status, singular, stateSimpleEffects) {
+  
+  if (identical(options$simpleFactor, "") | identical(options$moderatorFactorOne, ""))
```

Comment:
Please call a items within a list using "[[]]" rather than with $, for instance, 

options[["simpleFactor"]] 

This is more robust, which is necessary due to the fact that you have items that are named similar such as "moderatorFactorOne" and "moderatorFactorTwo".